include/ff_dir.h

```diff
@@ -64,6 +64,8 @@ typedef struct
     uint32_t ulAddrCurrentCluster;
     uint32_t ulDirCluster;
     uint16_t usCurrentItem;
+    /* Insert 2 bytes to make the next struct 32-bit aligned. */
+    uint16_t usFillerBytes;
```

Comment:
I assume that there will be very few instances of this struct. And we'll not waste a lot of memory with filler bytes.

Reply:
Yes indeed, there will be very few instances of this struct. It is used to list files and directories.